src/StoreKeeper/WooCommerce/B2C/Models/TaskModel.php

```diff
@@ -153,6 +155,50 @@ public static function countSuccessfulTasks(): int
         return TaskModel::count(['status = :status'], ['status' => TaskHandler::STATUS_SUCCESS]);
     }
 
+    public static function getTasksByCreatedDateTimeRange($start, $end, $limit, $order = 'DESC')
+    {
+        global $wpdb;
+
+        $select = static::getSelectHelper()
+            ->cols(['id', 'date_created'])
+            ->where('date_created >= :start')
+            ->where('date_created <= :end')
+            ->bindValue('start', $start)
+            ->bindValue('end', $end)
+            ->orderBy(['date_created '.$order])
+            ->limit($limit);
+
+        $query = static::prepareQuery($select);
+
+        $results = $wpdb->get_results($query, ARRAY_N);
+
+        return array_map(
+            function ($value) {
+                return [
+                    'id' => (int) $value[0],
+                    'date_created' => $value[1],
+                ];
+            },
+            $results
+        );
+    }
+
+    public static function getExecutionDurationSumByCreatedDateTimeRange($start, $end)
```

Comment:
```suggestion
    public static function getExecutionDurationSumByCreatedDateTimeRange($start, $end): float
```
Consistent return is better

Reply:
Changes implemented.